src/components/data_comps/datm/cime_config/namelist_definition_datm.xml

```diff
@@ -40,7 +40,8 @@
     CLM1PT		= Run with supplied single point data (force CLM)
     CORE2_NYF		= CORE2 normal year forcing (for forcing POP and CICE)
     CORE2_IAF		= CORE2 intra-annual year forcing (for forcing POP and CICE)
-    CORE_IAF_JRA		= JRA55 intra-annual year forcing (for forcing POP and CICE)
+    CORE_IAF_JRA	= JRA55 intra-annual year forcing (for forcing POP and CICE)
+    CORE_IAF_JRA_1p4_2018 = JRA55 intra-annual year forcing, v1.4, through 2018 (for forcing POP and CICE)
```

Comment:
**Is** there something specific regarding POP and CICE in this forcing file?

Reply:
These are the ATM and ROF forcing files used by POP and CICE.

Reply:
But I think they are also usable by the MPAS components as well

Reply:
DATM, and these new JRA options, can force any ocean or sea-ice model coupled with CIME.    At some point, those comments should be updated but don't have to do that now.